src/extern/turbomole.f90

```diff
@@ -269,9 +365,41 @@ subroutine external_turbomole(env,n,at,xyz,nel,nopen,extcode,extmode,grd,eel,g,d
 
 end subroutine external_turbomole
 
-!ccccccccccccccccccccccccccccccccc
-! TM
-!ccccccccccccccccccccccccccccccccc
+!----------------------------------------
+! create default TM control
+!----------------------------------------
```

Comment:
We could use this routine to create more customizable control files (see above) - but we can also do this at a later point.